controller/internal/enforcer/applicationproxy/tcp/tcp.go

```diff
@@ -444,7 +444,7 @@ func (p *Proxy) StartServerAuthStateMachine(ip net.IP, backendport int, upConn n
 				return isEncrypted, fmt.Errorf("reported rejected flow due to invalid token: %s", err)
 			}
 			tags := claims.T.Copy()
-			tags.AppendKeyValue(enforcerconstants.PortNumberLabelString, strconv.Itoa(backendport))
+			tags.AppendKeyValue(enforcerconstants.PortNumberLabelString, "TCP/"+strconv.Itoa(backendport))
 			report, packet := puContext.SearchRcvRules(tags)
```

Comment:
Can we create specific constants for that, rather than added it as a string here.